WordPress/Docs/WhiteSpace/OperatorSpacingStandard.xml

```diff
@@ -7,26 +7,33 @@
     <![CDATA[
     Always put one space on both sides of logical, comparison and concatenation operators.
     Always put one space after an assignment operator.
+    Most of these can be auto-fixed by PHP_CodeSniffer.
     ]]>
     </standard>
+
+
+
     <code_comparison>
         <code title="Valid: One space before and after an operator.">
         <![CDATA[
-if ( $a<em> === </em>$b<em> && </em>$b<em> === </em>$c ) {}
-if (<em> ! </em>$var ) {}
+        if ( $a === $b && $b === $c ) {}
```

Comment:
You shouldn't remove emphasis on errors. They serve a purpose (when you use HTML generator type you will see these bolded).

A lot of these fixes aren't valid...